tools/docker/22.04/Dockerfile

```diff
@@ -128,15 +128,34 @@ RUN apt-get update && \
     cd blaze && git checkout v3.4 && \
     cmake -DCMAKE_INSTALL_PREFIX=/usr/local/ && make install && cd ..
 
-# Install ASDF
+# Install Blosc2 (a prerequisite for ASDF)
+RUN mkdir src && \
+    (cd src && \
+    wget https://github.com/Blosc/c-blosc2/archive/refs/tags/v2.13.1.tar.gz && \
+    tar xzf v2.13.1.tar.gz && \
+    cd c-blosc2-2.13.1 && \
+    cmake -B build -S . \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DCMAKE_INSTALL_PREFIX=/usr/local \
+        -DBUILD_BENCHMARKS=OFF \
+        -DBUILD_EXAMPLES=OFF \
+        -DBUILD_FUZZERS=OFF \
+        -DBUILD_TESTS=OFF \
+        && \
+    cmake --build build --parallel $(nproc) && \
+    cmake --install build && \
+    true) && \
+    rm -rf src
+
+# Install ASDF (depends on Blosc2)
```

Comment:
At some point we installed ASDF twice into the Docker image. That is corrected on `develop` (methinks). Did you check this Dockerfile whether the problem is removed here as well?

Reply:
It's only installed once in the 22.04 container, and not at all in 18.04.